doc/concepts.rst

```diff
@@ -1193,7 +1193,7 @@ Foreign Sources, Archives and Objects
 Dune provides basic support for including foreign source files as well
 as archives of foreign object files into OCaml projects via the
 ``foreign_stubs`` and ``foreign_archives`` fields. Individual object
```

Comment:
So stubs and archives stay foreign whereas objects become extra?

Reply:
That's correct. The `foreign_objects` field was only added recently and we want to rename it before the next release to reflect that the binary object files are not necessarily foreign in the same sense that foreign stubs are foreign (ie. they are files in a non-ocaml source language). The `foreign_archives` field refers to archives of binary files which are also not necessarily foreign, but this field already exists in released versions of dune so we can't rename it easily.